docs/modules/jenkins/pages/webhook-proxy.adoc

```diff
@@ -47,13 +86,11 @@ The following environment variables are read by the proxy:
 | The base URL of the repository (e.g. your BitBucket host). This variable is set by the template and usually does not need to be modified.
 
 | TRIGGER_SECRET
-| The secret which protects the pipeline to be executed from outside. This variable is set by the template and usually does not need to be modified.
-|===
+| The secret which protects the pipeline to be executed from outside. This variable is set by the template and usually does not need to be modified. |
 
-Moreover, one can pass the following query parameters to the proxy:
-| Variable | Description |
-| -- | -- |
-| jenkinsfile_path | The path to the `Jenkinsfile`. By default, the `Jenkinsfile` is assumed to be in the root of the repository, therefore this value defaults to simply `Jenkinsfile`. |
+| ALLOWED_EXTERNAL_PROJECTS
+| Comma-separated list of external projects which the Webhook Proxy can deal with. By default, this is just the `opendevstack` project.
```

Comment:
When I go to View file this paragraph is not rendered as expected.

Reply:
Thanks! The problem was the trailing `|` on the previous line ... fixed in 82d2ea92e.